rpm/389-ds-base.spec.in

```diff
@@ -139,19 +139,7 @@ BuildRequires:    doxygen
 # For tests!
 BuildRequires:    libcmocka-devel
 # For lib389 and related components.
-BuildRequires:    python%{python3_pkgversion}
 BuildRequires:    python%{python3_pkgversion}-devel
-BuildRequires:    python%{python3_pkgversion}-setuptools
-BuildRequires:    python%{python3_pkgversion}-ldap
-BuildRequires:    python%{python3_pkgversion}-pyasn1
-BuildRequires:    python%{python3_pkgversion}-pyasn1-modules
-BuildRequires:    python%{python3_pkgversion}-dateutil
-BuildRequires:    python%{python3_pkgversion}-argcomplete
-BuildRequires:    python%{python3_pkgversion}-argparse-manpage
-BuildRequires:    python%{python3_pkgversion}-policycoreutils
-BuildRequires:    python%{python3_pkgversion}-libselinux
```

Comment:
The BuildRequires on python3-libselinux won't be generated. Either you need it and it needs to stay, or you don't need it (and in that case, why was it here?).